docs/contrib/GettingInvolved.md

```diff
@@ -3,11 +3,8 @@
 Thank you for taking the time to read this document and to contribute.
 The OpenROAD project will not reach all of its objectives without help!
 
-Possible ways to contribute:
+Possible ways to contribute in the scope of OpenROAD tool:
```

Comment:
Change to "Possible ways to contribute to the OpenROAD application:"

OpenROAD is the application name with underlying tools